packages/web/src/pages/edit-collection-page/utils.ts

```diff
@@ -0,0 +1,27 @@
+import { Collection } from '@audius/common/models'
+import { CollectionValues } from '@audius/common/schemas'
+import { Maybe } from '@audius/common/utils'
+
+import { removeNullable } from 'utils/typeUtils'
+
+// The edit collection form receives a tracks array, but we need to convert that to playlist_contents
+export const updatePlaylistContents = (
```

Comment:
so this is just reordering the existing playlist_contents array using tracks as source of truth of order only?

Reply:
order and deletes, yeah